servers/rendering/rendering_server_default.h

```diff
@@ -41,6 +41,7 @@
 #include "servers/rendering/renderer_compositor.h"
 #include "servers/rendering_server.h"
 #include "servers/server_wrap_mt_common.h"
+#include <cstdint>
```

Comment:
@akien-mga do we have a preference for including "core/typefs.h" rather than cstdint directly?